tuxemon/event/actions/print.py

```diff
@@ -0,0 +1,65 @@
+#
+# Tuxemon
+# Copyright (c) 2014-2017 William Edwards <shadowapex@gmail.com>,
+#                         Benjamin Bean <superman2k5@gmail.com>
+#
+# This file is part of Tuxemon
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+
+from __future__ import annotations
+import logging
+
+from tuxemon.event.eventaction import EventAction
+from typing import NamedTuple, Optional
+
+logger = logging.getLogger(__name__)
+
+
+class PrintActionParameters(NamedTuple):
+    variable: Optional[str]
+
+
+class PrintAction(EventAction[PrintActionParameters]):
+    """
+    Print the current value of a game variable.
```

Comment:
Please change to "Print the current value of a game variable to the console".  Some people might think it is printed to the game screen.